<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu, Ph.D." />


<title>Compare GWAS results on the same trait from two versions of input data files</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scratch</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Compare GWAS results on the same trait from
two versions of input data files</h1>
<h4 class="author">Xiang Zhu, Ph.D.</h4>
<h4 class="date">First created on 2025-04-04; Last modified on
2025-08-02</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-08-02
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>scratch/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250402code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250402)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250402code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250402)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree65cca22e46482d6ca731a4c266f7356f9a91ef41targetblank65cca22a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/scratch/tree/65cca22e46482d6ca731a4c266f7356f9a91ef41" target="_blank">65cca22</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree65cca22e46482d6ca731a4c266f7356f9a91ef41targetblank65cca22a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/scratch/tree/65cca22e46482d6ca731a4c266f7356f9a91ef41" target="_blank">65cca22</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/compare_dis_with_rep_h10.Rmd</code>) and HTML
(<code>docs/compare_dis_with_rep_h10.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/scratch/blob/65cca22e46482d6ca731a4c266f7356f9a91ef41/analysis/compare_dis_with_rep_h10.Rmd" target="_blank">65cca22</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-08-02
</td>
<td>
compare wgs-based discovery xiangzhu/hfpef#13 with wgs-based replication
xiangzhu/hfpef#18
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="eur-single-vs-eur-repeat-i2" class="section level2">
<h2>EUR Single vs EUR Repeat I2</h2>
<div id="overview-of-two-gwas-for-comparison" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of variants in the discovery GWAS:  82465759 </code></pre>
<pre><code>Number of variants in the replication GWAS:  21515596 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the replication GWAS:  1 </code></pre>
<pre><code>Number of GWAS loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of GWAS loci in the replication GWAS:  1 </code></pre>
<pre><code>Total number of variants shared by the discovery and replication:  21514492 </code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-discovery-gwas"
class="section level3">
<h3>Comparison across genome-wide significant variants from discovery
GWAS</h3>
<pre><code>Pearson correlation between dis_A1FREQ in new and rep_A1FREQ in old:
Estimate =  0.99976542 , 95% CI = [ 0.9997274 ,  0.99979813 ]

Regression coefficients for dis_A1FREQ (new ~ old):
               Estimate   Std. Error     t value     Pr(&gt;|t|)
(Intercept) 0.001722463 0.0003485570    4.941697 9.755478e-07
x           0.996018395 0.0008262545 1205.461954 0.000000e+00

Pearson correlation between dis_BETA in new and rep_BETA in old:
Estimate =  0.85809566 , 95% CI = [ 0.83697056 ,  0.87666719 ]

Regression coefficients for dis_BETA (new ~ old):
                Estimate   Std. Error   t value      Pr(&gt;|t|)
(Intercept) -0.001244643 0.0008141823 -1.528703  1.268017e-01
x            0.863389361 0.0197838435 43.641134 1.361007e-199

Pearson correlation between dis_SE in new and rep_SE in old:
Estimate =  0.99831761 , 95% CI = [ 0.99804519 ,  0.99855209 ]

Regression coefficients for dis_SE (new ~ old):
                 Estimate   Std. Error   t value    Pr(&gt;|t|)
(Intercept) -0.0003487191 1.603765e-05 -21.74378 4.83699e-80
x            0.2800943717 6.229298e-04 449.64036 0.00000e+00

Pearson correlation between dis_CHISQ in new and rep_CHISQ in old:
Estimate =  0.2507512 , 95% CI = [ 0.17915378 ,  0.31970657 ]

Regression coefficients for dis_CHISQ (new ~ old):
              Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 32.9757911  0.3856578 85.505306 0.000000e+00
x            0.7506294  0.1109657  6.764516 2.880184e-11

Pearson correlation between dis_LOG10P in new and rep_LOG10P in old:
Estimate =  0.23848182 , 95% CI = [ 0.16649328 ,  0.30794158 ]

Regression coefficients for dis_LOG10P (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 7.9543843 0.09879149 80.516899 0.000000e+00
x           0.5551698 0.08656918  6.413019 2.665723e-10

Pearson correlation between dis_ZSCORE in new and rep_ZSCORE in old:
Estimate =  0.88454804 , 95% CI = [ 0.86707916 ,  0.89984425 ]

Regression coefficients for dis_ZSCORE (new ~ old):
              Estimate Std. Error   t value      Pr(&gt;|t|)
(Intercept) -0.1591194 0.10642253 -1.495166  1.353336e-01
x            3.1424785 0.06345458 49.523274 4.597251e-228</code></pre>
</div>
<div id="assess-replication-at-the-variant-level"
class="section level3">
<h3>Assess replication at the variant-level</h3>
<pre><code>Total number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are present in the replication GWAS:  684 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of statistical significance:  191 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of both statistical significance and effect direction:  191 </code></pre>
</div>
<div id="assess-replication-at-the-locus-level" class="section level3">
<h3>Assess replication at the locus-level</h3>
<pre><code>Number of genome-wide significance loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of genome-wide significance loci in the discovery GWAS that are replicated in terms of statistical significance:  25 </code></pre>
</div>
<div id="visualization-of-comparison" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_dis_with_rep_h10.Rmd/unnamed-chunk-4-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="eur-single-vs-eur-repeat-i3" class="section level2">
<h2>EUR Single vs EUR Repeat I3</h2>
<div id="overview-of-two-gwas-for-comparison-1" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of variants in the discovery GWAS:  82465759 </code></pre>
<pre><code>Number of variants in the replication GWAS:  21542779 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the replication GWAS:  0 </code></pre>
<pre><code>Number of GWAS loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of GWAS loci in the replication GWAS:  0 </code></pre>
<pre><code>Total number of variants shared by the discovery and replication:  21541682 </code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-discovery-gwas-1"
class="section level3">
<h3>Comparison across genome-wide significant variants from discovery
GWAS</h3>
<pre><code>Pearson correlation between dis_A1FREQ in new and rep_A1FREQ in old:
Estimate =  0.99976358 , 95% CI = [ 0.99972527 ,  0.99979655 ]

Regression coefficients for dis_A1FREQ (new ~ old):
               Estimate   Std. Error     t value    Pr(&gt;|t|)
(Intercept) 0.002242816 0.0003495326    6.416616 2.60703e-10
x           0.995300068 0.0008288873 1200.766517 0.00000e+00

Pearson correlation between dis_BETA in new and rep_BETA in old:
Estimate =  0.84029889 , 95% CI = [ 0.81678565 ,  0.86102534 ]

Regression coefficients for dis_BETA (new ~ old):
                Estimate   Std. Error   t value      Pr(&gt;|t|)
(Intercept) -0.006120403 0.0008299366 -7.374544  4.797220e-13
x            1.026479467 0.0253584344 40.478819 1.641075e-183

Pearson correlation between dis_SE in new and rep_SE in old:
Estimate =  0.99800238 , 95% CI = [ 0.99767898 ,  0.99828075 ]

Regression coefficients for dis_SE (new ~ old):
                 Estimate   Std. Error   t value    Pr(&gt;|t|)
(Intercept) -0.0003452036 1.747119e-05 -19.75845 4.70451e-69
x            0.2774727448 6.725917e-04 412.54264 0.00000e+00

Pearson correlation between dis_CHISQ in new and rep_CHISQ in old:
Estimate =  0.3629979 , 95% CI = [ 0.29609007 ,  0.42636078 ]

Regression coefficients for dis_CHISQ (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 33.031483  0.2972632 111.11865 0.000000e+00
x            1.190779  0.1170451  10.17368 9.903533e-23

Pearson correlation between dis_LOG10P in new and rep_LOG10P in old:
Estimate =  0.36077689 , 95% CI = [ 0.29375665 ,  0.42426737 ]

Regression coefficients for dis_LOG10P (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 7.9060571  0.0761505 103.82147 0.000000e+00
x           0.8970484  0.0887983  10.10209 1.872141e-22

Pearson correlation between dis_ZSCORE in new and rep_ZSCORE in old:
Estimate =  0.83064503 , 95% CI = [ 0.80586037 ,  0.85252398 ]

Regression coefficients for dis_ZSCORE (new ~ old):
              Estimate Std. Error   t value      Pr(&gt;|t|)
(Intercept) -0.9204792 0.12189229 -7.551579  1.384621e-13
x            3.6142621 0.09277117 38.958894 1.359251e-175</code></pre>
</div>
<div id="assess-replication-at-the-variant-level-1"
class="section level3">
<h3>Assess replication at the variant-level</h3>
<pre><code>Total number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are present in the replication GWAS:  684 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of statistical significance:  79 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of both statistical significance and effect direction:  79 </code></pre>
</div>
<div id="assess-replication-at-the-locus-level-1"
class="section level3">
<h3>Assess replication at the locus-level</h3>
<pre><code>Number of genome-wide significance loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of genome-wide significance loci in the discovery GWAS that are replicated in terms of statistical significance:  25 </code></pre>
</div>
<div id="visualization-of-comparison-1" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_dis_with_rep_h10.Rmd/unnamed-chunk-7-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="eur-single-vs-csa-marginal-i2" class="section level2">
<h2>EUR Single vs CSA Marginal I2</h2>
<div id="overview-of-two-gwas-for-comparison-2" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of variants in the discovery GWAS:  82465759 </code></pre>
<pre><code>Number of variants in the replication GWAS:  16388175 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the replication GWAS:  0 </code></pre>
<pre><code>Number of GWAS loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of GWAS loci in the replication GWAS:  0 </code></pre>
<pre><code>Total number of variants shared by the discovery and replication:  14235003 </code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-discovery-gwas-2"
class="section level3">
<h3>Comparison across genome-wide significant variants from discovery
GWAS</h3>
<pre><code>Pearson correlation between dis_A1FREQ in new and rep_A1FREQ in old:
Estimate =  0.77401281 , 95% CI = [ 0.74183175 ,  0.80263698 ]

Regression coefficients for dis_A1FREQ (new ~ old):
             Estimate  Std. Error  t value      Pr(&gt;|t|)
(Intercept) 0.1051219 0.009865723 10.65527  1.341023e-24
x           0.7014343 0.022151230 31.66570 2.626333e-135

Pearson correlation between dis_BETA in new and rep_BETA in old:
Estimate =  0.65131727 , 95% CI = [ 0.60554901 ,  0.69279115 ]

Regression coefficients for dis_BETA (new ~ old):
               Estimate  Std. Error   t value     Pr(&gt;|t|)
(Intercept) -0.01213113 0.001089859 -11.13091 1.583724e-26
x            0.36673821 0.016494258  22.23430 1.743029e-82

Pearson correlation between dis_SE in new and rep_SE in old:
Estimate =  0.80188823 , 95% CI = [ 0.77316912 ,  0.82732529 ]

Regression coefficients for dis_SE (new ~ old):
               Estimate   Std. Error  t value      Pr(&gt;|t|)
(Intercept) 0.003427813 8.828014e-05 38.82881 9.328398e-174
x           0.059340735 1.706856e-03 34.76610 3.119175e-152

Pearson correlation between dis_CHISQ in new and rep_CHISQ in old:
Estimate =  0.28567529 , 95% CI = [ 0.21473582 ,  0.35361631 ]

Regression coefficients for dis_CHISQ (new ~ old):
             Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 33.548348  0.3017629 111.174529 0.000000e+00
x            1.136342  0.1471596   7.721834 4.179377e-14

Pearson correlation between dis_LOG10P in new and rep_LOG10P in old:
Estimate =  0.28787258 , 95% CI = [ 0.21701843 ,  0.35570934 ]

Regression coefficients for dis_LOG10P (new ~ old):
            Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 8.036003 0.07789703 103.161869 0.000000e+00
x           0.837605 0.10757045   7.786571 2.617065e-14

Pearson correlation between dis_ZSCORE in new and rep_ZSCORE in old:
Estimate =  0.53807271 , 95% CI = [ 0.48210183 ,  0.58966933 ]

Regression coefficients for dis_ZSCORE (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) -2.101785  0.1808571 -11.62125 1.424116e-28
x            2.570716  0.1554630  16.53587 8.937085e-52</code></pre>
</div>
<div id="assess-replication-at-the-variant-level-2"
class="section level3">
<h3>Assess replication at the variant-level</h3>
<pre><code>Total number of genome-wide significant variants in the discovery GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are present in the replication GWAS:  673 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of statistical significance:  70 </code></pre>
<pre><code>Number of genome-wide significant variants in the discovery GWAS that are replicated in terms of both statistical significance and effect direction:  70 </code></pre>
</div>
<div id="assess-replication-at-the-locus-level-2"
class="section level3">
<h3>Assess replication at the locus-level</h3>
<pre><code>Number of genome-wide significance loci in the discovery GWAS:  25 </code></pre>
<pre><code>Number of genome-wide significance loci in the discovery GWAS that are replicated in terms of statistical significance:  25 </code></pre>
</div>
<div id="visualization-of-comparison-2" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_dis_with_rep_h10.Rmd/unnamed-chunk-10-1.png" width="1440" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.5.1 (2025-06-13)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.1

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Los_Angeles
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] xzTools_0.0.0.9000 patchwork_1.3.1    ggplot2_3.5.2      dplyr_1.1.4       
[5] data.table_1.17.8  workflowr_1.7.1   

loaded via a namespace (and not attached):
 [1] sass_0.4.10        generics_0.1.4     lattice_0.22-7     stringi_1.8.7     
 [5] digest_0.6.37      magrittr_2.0.3     evaluate_1.0.4     grid_4.5.1        
 [9] RColorBrewer_1.1-3 fastmap_1.2.0      Matrix_1.7-3       R.oo_1.27.1       
[13] rprojroot_2.0.4    jsonlite_2.0.0     processx_3.8.6     R.utils_2.13.0    
[17] whisker_0.4.1      ps_1.9.1           promises_1.3.3     mgcv_1.9-3        
[21] httr_1.4.7         purrr_1.1.0        scales_1.4.0       jquerylib_0.1.4   
[25] cli_3.6.5          rlang_1.1.6        R.methodsS3_1.8.2  splines_4.5.1     
[29] withr_3.0.2        cachem_1.1.0       yaml_2.3.10        tools_4.5.1       
[33] httpuv_1.6.16      vctrs_0.6.5        R6_2.6.1           lifecycle_1.0.4   
[37] git2r_0.36.2       stringr_1.5.1      fs_1.6.6           pkgconfig_2.0.3   
[41] callr_3.7.6        pillar_1.11.0      bslib_0.9.0        later_1.4.2       
[45] gtable_0.3.6       glue_1.8.0         Rcpp_1.1.0         xfun_0.52         
[49] tibble_3.3.0       tidyselect_1.2.1   rstudioapi_0.17.1  knitr_1.50        
[53] farver_2.1.2       nlme_3.1-168       htmltools_0.5.8.1  labeling_0.4.3    
[57] rmarkdown_2.29     compiler_4.5.1     getPass_0.2-4     </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
