<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu, Ph.D." />


<title>Compare GWAS results on the same trait from two versions of input data files</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scratch</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Compare GWAS results on the same trait from
two versions of input data files</h1>
<h4 class="author">Xiang Zhu, Ph.D.</h4>
<h4 class="date">First created on 2025-04-04; Last modified on
2025-04-08</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-08
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>scratch/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250402code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250402)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250402code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250402)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree7d8a4747ec3be2e530af5a2cb2e9fc6adca6a106targetblank7d8a474a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/scratch/tree/7d8a4747ec3be2e530af5a2cb2e9fc6adca6a106" target="_blank">7d8a474</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree7d8a4747ec3be2e530af5a2cb2e9fc6adca6a106targetblank7d8a474a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/scratch/tree/7d8a4747ec3be2e530af5a2cb2e9fc6adca6a106" target="_blank">7d8a474</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/compare_not_with_dcf_04.Rmd</code>) and HTML
(<code>docs/compare_not_with_dcf_04.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/scratch/blob/7d8a4747ec3be2e530af5a2cb2e9fc6adca6a106/analysis/compare_not_with_dcf_04.Rmd" target="_blank">7d8a474</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-04-08
</td>
<td>
wflow_publish("analysis/compare_not_with_dcf_04.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="overview-of-two-gwas-for-comparison" class="section level2">
<h2>Overview of two GWAS for comparison</h2>
<pre><code>Total number of rows in new data frame:  49078469 </code></pre>
<pre><code>Total number of rows in old data frame:  49563937 </code></pre>
<pre><code>Total number of rows after joining two data frames:  49074445 </code></pre>
<pre><code>Number of unique variant IDs shared by two data frames:  48850276 </code></pre>
<pre><code>Number of unique variant IDs after joining two data frames:  48850272 </code></pre>
<pre><code>Look at the GWAS results 4 shared variant IDs outside the joint data frame: </code></pre>
<pre><code>   CHROM    GENPOS          ID ALLELE0 ALLELE1  dcf_A1FREQ dcf_INFO dcf_N
   &lt;int&gt;     &lt;int&gt;      &lt;char&gt;  &lt;char&gt;  &lt;char&gt;       &lt;num&gt;    &lt;num&gt; &lt;int&gt;
1:     5 134668743 rs907339653       C       T 1.18249e-04 0.802187 44008
2:     6  45149850 rs751965194       G       A 8.09122e-05 0.812812 44008
3:    12  38796898 rs566175979       C       G 5.68079e-05 0.866659 44008
4:    13  91377272 rs113968307       T       C 1.97829e-03 0.800423 44008
     dcf_BETA    dcf_SE  dcf_CHISQ dcf_LOG10P
        &lt;num&gt;     &lt;num&gt;      &lt;num&gt;      &lt;num&gt;
1:  0.2925350 0.3345550 0.76457700  0.4180500
2:  0.4613870 0.4014800 1.32070000  0.6012500
3: -0.0576274 0.4642020 0.01541150  0.0451778
4:  0.0049679 0.0817123 0.00369633  0.0215818</code></pre>
<pre><code>   CHROM    GENPOS          ID ALLELE0 ALLELE1  not_A1FREQ not_INFO not_N
   &lt;int&gt;     &lt;int&gt;      &lt;char&gt;  &lt;char&gt;  &lt;char&gt;       &lt;num&gt;    &lt;num&gt; &lt;int&gt;
1:     5 134668743 rs907339653       C       A 7.12437e-05 0.817927 44008
2:     6  45149850 rs751965194       G       C 7.89072e-05 0.810722 44008
3:    12  38796898 rs566175979       C       T 2.62831e-04 0.800463 44008
4:    13  91377272 rs113968307       T       A 6.07733e-05 0.829068 44008
    not_BETA   not_SE  not_CHISQ not_LOG10P
       &lt;num&gt;    &lt;num&gt;      &lt;num&gt;      &lt;num&gt;
1: 0.0178312 0.432103 0.00170289  0.0145359
2: 0.2327440 0.403439 0.33281500  0.2487160
3: 0.1599200 0.219538 0.53062000  0.3312910
4: 0.1511860 0.445147 0.11535000  0.1342250</code></pre>
</div>
<div id="comparison-across-all-variants" class="section level2">
<h2>Comparison across all variants</h2>
<pre><code>A1FREQ values are identical between new (dcf) and old (not) data frames. </code></pre>
<pre><code>INFO values are identical between new (dcf) and old (not) data frames. </code></pre>
<pre><code>N values are identical between new (dcf) and old (not) data frames. </code></pre>
<pre><code>Pearson correlation between dcf_BETA in new and not_BETA in old:
Estimate = 0.99370529, 95% CI = [0.99370178, 0.9937088]

Regression coefficients for dcf_BETA (new ~ old):
                 Estimate   Std. Error     t value     Pr(&gt;|t|)
(Intercept) -6.013699e-05 4.236408e-06   -14.19528 9.801642e-46
x            1.021997e+00 1.644685e-05 62139.37337 0.000000e+00

Pearson correlation between dcf_SE in new and not_SE in old:
Estimate = 0.99988433, 95% CI = [0.99988427, 0.9998844]

Regression coefficients for dcf_SE (new ~ old):
                Estimate   Std. Error     t value Pr(&gt;|t|)
(Intercept) 6.890656e-05 5.742487e-07    119.9943        0
x           1.027336e+00 2.230729e-06 460538.1538        0

Pearson correlation between dcf_CHISQ in new and not_CHISQ in old:
Estimate = 0.99221774, 95% CI = [0.9922134, 0.99222208]

Regression coefficients for dcf_CHISQ (new ~ old):
              Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.01730581 3.766795e-05   459.4306        0
x           0.98530715 1.765055e-05 55823.0410        0

Pearson correlation between dcf_LOG10P in new and not_LOG10P in old:
Estimate = 0.99028837, 95% CI = [0.99028296, 0.99029378]

Regression coefficients for dcf_LOG10P (new ~ old):
               Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.006733461 1.353613e-05   497.4435        0
x           0.986299266 1.976621e-05 49898.2517        0

Pearson correlation between dcf_ZSCORE in new and not_ZSCORE in old:
Estimate = 0.99409643, 95% CI = [0.99409313, 0.99409972]

Regression coefficients for dcf_ZSCORE (new ~ old):
                 Estimate   Std. Error      t value     Pr(&gt;|t|)
(Intercept) -0.0001226044 1.591268e-05    -7.704823 1.310277e-14
x            0.9949563063 1.550166e-05 64183.846702 0.000000e+00</code></pre>
</div>
<div id="comparison-across-genome-wide-significant-variants-from-dcf"
class="section level2">
<h2>Comparison across genome-wide significant variants from
<code>dcf</code></h2>
<pre><code>Pearson correlation between dcf_BETA in new and not_BETA in old:
Estimate = 0.99984399, 95% CI = [0.99983696, 0.99985072]

Regression coefficients for dcf_BETA (new ~ old):
                 Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) -0.0007466435 1.837381e-05  -40.63629        0
x            1.0262739868 2.038645e-04 5034.09815        0

Pearson correlation between dcf_SE in new and not_SE in old:
Estimate = 0.99999477, 95% CI = [0.99999453, 0.99999499]

Regression coefficients for dcf_SE (new ~ old):
                Estimate   Std. Error     t value      Pr(&gt;|t|)
(Intercept) 1.501284e-05 3.763683e-07    39.88868 3.427873e-317
x           1.024950e+00 3.728397e-05 27490.35446  0.000000e+00

Pearson correlation between dcf_CHISQ in new and not_CHISQ in old:
Estimate = 0.99887463, 95% CI = [0.99882395, 0.99892314]

Regression coefficients for dcf_CHISQ (new ~ old):
             Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 2.1815085 0.0432147733   50.48062        0
x           0.9509099 0.0005077004 1872.97447        0

Pearson correlation between dcf_LOG10P in new and not_LOG10P in old:
Estimate = 0.99887287, 95% CI = [0.9988221, 0.99892145]

Regression coefficients for dcf_LOG10P (new ~ old):
             Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.5205280 0.0098773017   52.69941        0
x           0.9509588 0.0005081247 1871.50682        0

Pearson correlation between dcf_ZSCORE in new and not_ZSCORE in old:
Estimate = 0.99982723, 95% CI = [0.99981944, 0.99983468]

Regression coefficients for dcf_ZSCORE (new ~ old):
              Estimate   Std. Error    t value      Pr(&gt;|t|)
(Intercept) -0.0455024 0.0018046508  -25.21396 5.423609e-135
x            0.9927400 0.0002075304 4783.58931  0.000000e+00</code></pre>
</div>
<div id="comparison-across-genome-wide-significant-variants-from-not"
class="section level2">
<h2>Comparison across genome-wide significant variants from
<code>not</code></h2>
<pre><code>Pearson correlation between dcf_BETA in new and not_BETA in old:
Estimate = 0.99984242, 95% CI = [0.99983524, 0.99984928]

Regression coefficients for dcf_BETA (new ~ old):
                 Estimate   Std. Error   t value    Pr(&gt;|t|)
(Intercept) -0.0007033148 1.883023e-05  -37.3503 6.5613e-281
x            1.0253916057 2.068172e-04 4957.9619  0.0000e+00

Pearson correlation between dcf_SE in new and not_SE in old:
Estimate = 0.99999497, 95% CI = [0.99999474, 0.99999519]

Regression coefficients for dcf_SE (new ~ old):
                Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 1.533323e-05 3.763080e-07    40.7465        0
x           1.024895e+00 3.693622e-05 27747.7053        0

Pearson correlation between dcf_CHISQ in new and not_CHISQ in old:
Estimate = 0.99881954, 95% CI = [0.99876581, 0.99887093]

Regression coefficients for dcf_CHISQ (new ~ old):
             Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 1.9542899 0.0452267214   43.21096        0
x           0.9529979 0.0005264955 1810.07806        0

Pearson correlation between dcf_LOG10P in new and not_LOG10P in old:
Estimate = 0.9988165, 95% CI = [0.99876264, 0.99886802]

Regression coefficients for dcf_LOG10P (new ~ old):
             Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.4678152 0.0103409528   45.23908        0
x           0.9530918 0.0005272263 1807.74715        0

Pearson correlation between dcf_ZSCORE in new and not_ZSCORE in old:
Estimate = 0.9998265, 95% CI = [0.9998186, 0.99983405]

Regression coefficients for dcf_ZSCORE (new ~ old):
               Estimate   Std. Error   t value     Pr(&gt;|t|)
(Intercept) -0.03679875 0.0018368606  -20.0335 4.383107e-87
x            0.99160028 0.0002098642 4724.9619 0.000000e+00</code></pre>
</div>
<div id="overlap-of-genome-wide-significant-variants-between-two-gwas"
class="section level2">
<h2>Overlap of genome-wide significant variants between two GWAS</h2>
<pre><code>              DCF p &lt;= 5e-8 DCF p &gt; 5e-8
NOT p &lt;= 5e-8          7632          119
NOT p &gt; 5e-8            279     49066415</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value &lt; 2.2e-16
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 8.402051e+03 4.503600e+15
sample estimates:
odds ratio 
4.5036e+15 </code></pre>
</div>
<div id="visualization-of-comparison" class="section level2">
<h2>Visualization of comparison</h2>
<p><img src="figure/compare_not_with_dcf_04.Rmd/unnamed-chunk-3-1.png" width="1440" style="display: block; margin: auto;" /><img src="figure/compare_not_with_dcf_04.Rmd/unnamed-chunk-3-2.png" width="1440" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.4.3 (2025-02-28)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.4

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Los_Angeles
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] xzTools_0.0.0.9000 patchwork_1.3.0    ggplot2_3.5.1      dplyr_1.1.4       
[5] data.table_1.17.0  workflowr_1.7.1   

loaded via a namespace (and not attached):
 [1] sass_0.4.9        generics_0.1.3    lattice_0.22-6    stringi_1.8.7    
 [5] digest_0.6.37     magrittr_2.0.3    evaluate_1.0.3    grid_4.4.3       
 [9] fastmap_1.2.0     Matrix_1.7-2      R.oo_1.27.0       rprojroot_2.0.4  
[13] jsonlite_1.9.1    processx_3.8.6    R.utils_2.12.3    whisker_0.4.1    
[17] ps_1.9.0          promises_1.3.2    mgcv_1.9-1        httr_1.4.7       
[21] scales_1.3.0      jquerylib_0.1.4   cli_3.6.4         rlang_1.1.5      
[25] R.methodsS3_1.8.2 splines_4.4.3     munsell_0.5.1     withr_3.0.2      
[29] cachem_1.1.0      yaml_2.3.10       tools_4.4.3       colorspace_2.1-1 
[33] httpuv_1.6.15     vctrs_0.6.5       R6_2.6.1          lifecycle_1.0.4  
[37] git2r_0.35.0      stringr_1.5.1     fs_1.6.5          pkgconfig_2.0.3  
[41] callr_3.7.6       pillar_1.10.2     bslib_0.9.0       later_1.4.1      
[45] gtable_0.3.6      glue_1.8.0        Rcpp_1.0.14       xfun_0.51        
[49] tibble_3.2.1      tidyselect_1.2.1  rstudioapi_0.17.1 knitr_1.50       
[53] farver_2.1.2      nlme_3.1-167      htmltools_0.5.8.1 labeling_0.4.3   
[57] rmarkdown_2.29    compiler_4.4.3    getPass_0.2-4    </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
