<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu, Ph.D." />


<title>Compare GWAS results on the same trait from two versions of input data files</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scratch</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Compare GWAS results on the same trait from
two versions of input data files</h1>
<h4 class="author">Xiang Zhu, Ph.D.</h4>
<h4 class="date">First created on 2025-04-04; Last modified on
2025-07-27</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-07-27
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>scratch/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250402code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250402)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250402code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250402)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree4229b5f93d5a6a130903418f56cd30edd7e15ea0targetblank4229b5fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/scratch/tree/4229b5f93d5a6a130903418f56cd30edd7e15ea0" target="_blank">4229b5f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree4229b5f93d5a6a130903418f56cd30edd7e15ea0targetblank4229b5fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/scratch/tree/4229b5f93d5a6a130903418f56cd30edd7e15ea0" target="_blank">4229b5f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/compare_wgs_with_gel_b06_rep.Rmd</code>) and HTML
(<code>docs/compare_wgs_with_gel_b06_rep.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/scratch/blob/4229b5f93d5a6a130903418f56cd30edd7e15ea0/analysis/compare_wgs_with_gel_b06_rep.Rmd" target="_blank">4229b5f</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-07-27
</td>
<td>
compare wgs-based xiangzhu/brain2gene#14 with gel-based
xiangzhu/brain2gene#2 results in replication xiangzhu/brain2gene#7
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="ukb-wgs-dcf-i2-vs-ukb-gel-not-i2" class="section level2">
<h2>UKB-WGS DCF I2 vs UKB-GEL NOT I2</h2>
<div id="overview-of-two-gwas-for-comparison" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of rows in the new data frame:  21639074 </code></pre>
<pre><code>Number of rows in the old data frame:  16663603 </code></pre>
<pre><code>Number of genome-wide significant variants in the new GWAS:  0 </code></pre>
<pre><code>Number of genome-wide significant variants in the old GWAS:  0 </code></pre>
<pre><code>Number of GWAS loci in the new GWAS:  0 </code></pre>
<pre><code>Number of GWAS loci in the old GWAS:  0 </code></pre>
<pre><code>Relaxed genome-wide significant threshold:  4.30103 </code></pre>
<pre><code>Total number of rows after joining two data frames:  16443698 </code></pre>
</div>
<div id="comparison-across-all-variants" class="section level3">
<h3>Comparison across all variants</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99989306 , 95% CI = [ 0.99989295 ,  0.99989316 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error      t value      Pr(&gt;|t|)
(Intercept) 3.167999e-05 9.858724e-07     32.13396 1.503938e-226
x           9.998096e-01 3.606172e-06 277249.51442  0.000000e+00

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.71982369 , 95% CI = [ 0.71959071 ,  0.7200565 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate   Std. Error    t value     Pr(&gt;|t|)
(Intercept) 0.0006263792 3.236582e-05   19.35311 1.923208e-83
x           0.7291062614 1.733894e-04 4205.02113 0.000000e+00

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99360806 , 95% CI = [ 0.9936019 ,  0.99361422 ]

Regression coefficients for new_SE (new ~ old):
               Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.001494194 5.262972e-06   283.9069        0
x           0.978246321 2.740751e-05 35692.6424        0

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.54244695 , 95% CI = [ 0.54210575 ,  0.54278797 ]

Regression coefficients for new_CHISQ (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.4593532 0.0003595740 1277.493        0
x           0.5693851 0.0002174578 2618.371        0

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.53444521 , 95% CI = [ 0.53409985 ,  0.5347904 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.2018532 0.0001285031 1570.804        0
x           0.5552393 0.0002165401 2564.141        0

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.73587535 , 95% CI = [ 0.73565367 ,  0.73609688 ]

Regression coefficients for new_ZSCORE (new ~ old):
               Estimate   Std. Error    t value     Pr(&gt;|t|)
(Intercept) 0.002014317 0.0001672480   12.04389 2.089347e-33
x           0.754150121 0.0001711266 4406.97152 0.000000e+00</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-ukb-wgs"
class="section level3">
<h3>Comparison across genome-wide significant variants from UKB-WGS</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99998637 , 95% CI = [ 0.99998444 ,  0.99998807 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error     t value   Pr(&gt;|t|)
(Intercept) 0.0001072642 4.878328e-05    2.198791 0.02815508
x           0.9992618317 1.767521e-04 5653.465338 0.00000000

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.97444285 , 95% CI = [ 0.97086324 ,  0.97758768 ]

Regression coefficients for new_BETA (new ~ old):
              Estimate  Std. Error    t value     Pr(&gt;|t|)
(Intercept) 0.02236304 0.005752859   3.887292 0.0001090679
x           1.29167015 0.010089384 128.022693 0.0000000000

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99546768 , 95% CI = [ 0.99482515 ,  0.99603059 ]

Regression coefficients for new_SE (new ~ old):
               Estimate   Std. Error    t value   Pr(&gt;|t|)
(Intercept) 0.001311684 0.0005720066   2.293127 0.02207816
x           0.974774967 0.0031553730 308.925428 0.00000000

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.31667334 , 95% CI = [ 0.25569455 ,  0.37514233 ]

Regression coefficients for new_CHISQ (new ~ old):
              Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 16.8152008 0.15684217 107.210966 0.000000e+00
x            0.1390244 0.01410989   9.852975 8.713261e-22

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.31549104 , 95% CI = [ 0.25446648 ,  0.3740131 ]

Regression coefficients for new_LOG10P (new ~ old):
            Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 4.330488 0.04108558 105.401643 0.000000e+00
x           0.133103 0.01356517   9.812114 1.255516e-21

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.98125198 , 95% CI = [ 0.97861575 ,  0.98356592 ]

Regression coefficients for new_ZSCORE (new ~ old):
             Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 0.1407532 0.02800490   5.026018 6.078381e-07
x           1.3086931 0.00870955 150.259557 0.000000e+00</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-ukb-gel"
class="section level3">
<h3>Comparison across genome-wide significant variants from UKB-GEL</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99997401 , 95% CI = [ 0.99996826 ,  0.99997871 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error     t value  Pr(&gt;|t|)
(Intercept) 0.0001221882 8.032995e-05    1.521079 0.1290588
x           1.0004767851 3.671629e-04 2724.885534 0.0000000

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.96433237 , 95% CI = [ 0.95661622 ,  0.9706967 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate  Std. Error    t value      Pr(&gt;|t|)
(Intercept) -0.004389944 0.009479736 -0.4630872  6.435629e-01
x            0.746467410 0.010428817 71.5773780 6.360252e-225

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99404043 , 95% CI = [ 0.99272737 ,  0.995117 ]

Regression coefficients for new_SE (new ~ old):
               Estimate  Std. Error     t value  Pr(&gt;|t|)
(Intercept) 0.001126392 0.001171898   0.9611689 0.3370689
x           0.982357312 0.005483355 179.1525997 0.0000000

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.2051684 , 95% CI = [ 0.10781249 ,  0.29862671 ]

Regression coefficients for new_CHISQ (new ~ old):
             Estimate Std. Error    t value     Pr(&gt;|t|)
(Intercept) 0.1563583  2.7310137 0.05725286 9.543734e-01
x           0.6188160  0.1502515 4.11853439 4.667007e-05

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.20253142 , 95% CI = [ 0.10509236 ,  0.2961187 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 0.1545358  0.7286405 0.2120878 8.321504e-01
x           0.6308514  0.1552551 4.0633203 5.863097e-05

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.97486397 , 95% CI = [ 0.96939067 ,  0.97936884 ]

Regression coefficients for new_ZSCORE (new ~ old):
                Estimate  Std. Error    t value      Pr(&gt;|t|)
(Intercept) -0.004754959 0.038218904 -0.1244138  9.010524e-01
x            0.772246628 0.008983287 85.9648180 8.228024e-254</code></pre>
</div>
<div id="overlap-of-genome-wide-significant-variants-between-two-gwas"
class="section level3">
<h3>Overlap of genome-wide significant variants between two GWAS</h3>
<pre><code>                  UKB-WGS p &lt; 5e-8 UKB-WGS p &gt;= 5e-8
UKB-GEL p &lt; 5e-8                67               321
UKB-GEL p &gt;= 5e-8              806          16442504</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value &lt; 2.2e-16
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 3062.43 8192.00
sample estimates:
odds ratio 
  3801.874 </code></pre>
</div>
<div id="visualization-of-comparison" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_wgs_with_gel_b06_rep.Rmd/unnamed-chunk-3-1.png" width="1440" style="display: block; margin: auto;" /><img src="figure/compare_wgs_with_gel_b06_rep.Rmd/unnamed-chunk-3-2.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ukb-wgs-dcf-i3-vs-ukb-gel-not-i3" class="section level2">
<h2>UKB-WGS DCF I3 vs UKB-GEL NOT I3</h2>
<div id="overview-of-two-gwas-for-comparison-1" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of rows in the new data frame:  21762110 </code></pre>
<pre><code>Number of rows in the old data frame:  16746315 </code></pre>
<pre><code>Number of genome-wide significant variants in the new GWAS:  4 </code></pre>
<pre><code>Number of genome-wide significant variants in the old GWAS:  0 </code></pre>
<pre><code>Number of GWAS loci in the new GWAS:  3 </code></pre>
<pre><code>Number of GWAS loci in the old GWAS:  0 </code></pre>
<pre><code>Relaxed genome-wide significant threshold:  4.30103 </code></pre>
<pre><code>Total number of rows after joining two data frames:  16524258 </code></pre>
</div>
<div id="comparison-across-all-variants-1" class="section level3">
<h3>Comparison across all variants</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99989244 , 95% CI = [ 0.99989234 ,  0.99989255 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error      t value      Pr(&gt;|t|)
(Intercept) 3.218857e-05 9.841717e-07     32.70626 1.294809e-234
x           9.998005e-01 3.607631e-06 277134.89698  0.000000e+00

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.75205571 , 95% CI = [ 0.75184618 ,  0.75226508 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate   Std. Error     t value     Pr(&gt;|t|)
(Intercept) 0.0002745594 3.037957e-05    9.037635 1.601143e-19
x           0.7483144988 1.613334e-04 4638.311950 0.000000e+00

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99365304 , 95% CI = [ 0.99364693 ,  0.99365913 ]

Regression coefficients for new_SE (new ~ old):
               Estimate   Std. Error    t value Pr(&gt;|t|)
(Intercept) 0.001415827 5.183447e-06   273.1438        0
x           0.965463794 2.688736e-05 35907.7256        0

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.58860984 , 95% CI = [ 0.58829464 ,  0.58892486 ]

Regression coefficients for new_CHISQ (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.4086614 0.0003423366 1193.742        0
x           0.6103016 0.0002062014 2959.735        0

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.58048039 , 95% CI = [ 0.58016061 ,  0.58079999 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.1806014 0.0001226426 1472.582        0
x           0.5971780 0.0002060753 2897.863        0

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.76617651 , 95% CI = [ 0.76597732 ,  0.76637556 ]

Regression coefficients for new_ZSCORE (new ~ old):
                Estimate   Std. Error      t value  Pr(&gt;|t|)
(Intercept) 0.0001505039 0.0001575842    0.9550695 0.3395426
x           0.7800943423 0.0001609602 4846.5050771 0.0000000</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-ukb-wgs-1"
class="section level3">
<h3>Comparison across genome-wide significant variants from UKB-WGS</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99999359 , 95% CI = [ 0.99999256 ,  0.99999448 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error     t value   Pr(&gt;|t|)
(Intercept) 9.889059e-05 5.071385e-05    1.949972 0.05158232
x           9.995903e-01 1.359228e-04 7354.103617 0.00000000

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.97233051 , 95% CI = [ 0.96795559 ,  0.9761154 ]

Regression coefficients for new_BETA (new ~ old):
               Estimate  Std. Error     t value  Pr(&gt;|t|)
(Intercept) 0.001029272 0.006858061   0.1500821 0.8807435
x           1.291571522 0.011787680 109.5696161 0.0000000

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99442299 , 95% CI = [ 0.99352973 ,  0.99519323 ]

Regression coefficients for new_SE (new ~ old):
                Estimate   Std. Error     t value  Pr(&gt;|t|)
(Intercept) 0.0006878822 0.0007235241   0.9507385 0.3420686
x           0.9760472002 0.0039322604 248.2153029 0.0000000

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.37537735 , 95% CI = [ 0.30965268 ,  0.43753209 ]

Regression coefficients for new_CHISQ (new ~ old):
              Estimate Std. Error  t value     Pr(&gt;|t|)
(Intercept) 16.3260096 0.17992305 90.73884 0.000000e+00
x            0.1617527 0.01517178 10.66142 1.116705e-24

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.37143781 , 95% CI = [ 0.30550792 ,  0.43382319 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate Std. Error  t value     Pr(&gt;|t|)
(Intercept) 4.2118999 0.04694378 89.72221 0.000000e+00
x           0.1543485 0.01465588 10.53151 3.688964e-24

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.98238118 , 95% CI = [ 0.97957895 ,  0.98480184 ]

Regression coefficients for new_ZSCORE (new ~ old):
              Estimate  Std. Error    t value  Pr(&gt;|t|)
(Intercept) 0.03828627 0.030248699   1.265716 0.2060401
x           1.25826799 0.009093026 138.377254 0.0000000</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-ukb-gel-1"
class="section level3">
<h3>Comparison across genome-wide significant variants from UKB-GEL</h3>
<pre><code>Pearson correlation between new_A1FREQ in new and old_A1FREQ in old:
Estimate =  0.99999094 , 95% CI = [ 0.9999892 ,  0.9999924 ]

Regression coefficients for new_A1FREQ (new ~ old):
                Estimate   Std. Error     t value  Pr(&gt;|t|)
(Intercept) 5.136629e-05 5.070547e-05    1.013033 0.3115365
x           1.001191e+00 1.909723e-04 5242.597966 0.0000000

Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  0.9786611 , 95% CI = [ 0.97461101 ,  0.98207098 ]

Regression coefficients for new_BETA (new ~ old):
               Estimate  Std. Error    t value   Pr(&gt;|t|)
(Intercept) -0.01341061 0.005707771  -2.349536 0.01918645
x            0.78121922 0.007350174 106.285805 0.00000000

Pearson correlation between new_SE in new and old_SE in old:
Estimate =  0.99462384 , 95% CI = [ 0.99359364 ,  0.99548874 ]

Regression coefficients for new_SE (new ~ old):
                 Estimate   Std. Error     t value  Pr(&gt;|t|)
(Intercept) -0.0003922536 0.0008390205  -0.4675138 0.6403369
x            0.9969101329 0.0046510303 214.3417848 0.0000000

Pearson correlation between new_CHISQ in new and old_CHISQ in old:
Estimate =  0.22963552 , 95% CI = [ 0.144862 ,  0.31106228 ]

Regression coefficients for new_CHISQ (new ~ old):
             Estimate Std. Error  t value     Pr(&gt;|t|)
(Intercept) 1.9209098  1.8575756 1.034095 3.015936e-01
x           0.5293219  0.1005316 5.265231 2.086183e-07

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.22539382 , 95% CI = [ 0.14047981 ,  0.30701626 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate Std. Error  t value     Pr(&gt;|t|)
(Intercept) 0.6556651  0.4911433 1.334977 1.824938e-01
x           0.5325765  0.1031582 5.162718 3.523941e-07

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  0.98379982 , 95% CI = [ 0.98071555 ,  0.9863942 ]

Regression coefficients for new_ZSCORE (new ~ old):
               Estimate  Std. Error    t value   Pr(&gt;|t|)
(Intercept) -0.04695542 0.027433948  -1.711581 0.08759658
x            0.78366617 0.006399094 122.465170 0.00000000</code></pre>
</div>
<div id="overlap-of-genome-wide-significant-variants-between-two-gwas-1"
class="section level3">
<h3>Overlap of genome-wide significant variants between two GWAS</h3>
<pre><code>                  UKB-WGS p &lt; 5e-8 UKB-WGS p &gt;= 5e-8
UKB-GEL p &lt; 5e-8                68               432
UKB-GEL p &gt;= 5e-8              627          16523131</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value &lt; 2.2e-16
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 3062.43 8192.00
sample estimates:
odds ratio 
   3676.72 </code></pre>
</div>
<div id="visualization-of-comparison-1" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_wgs_with_gel_b06_rep.Rmd/unnamed-chunk-6-1.png" width="1440" style="display: block; margin: auto;" /><img src="figure/compare_wgs_with_gel_b06_rep.Rmd/unnamed-chunk-6-2.png" width="1440" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.5.1 (2025-06-13)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.1

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Los_Angeles
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] xzTools_0.0.0.9000 patchwork_1.3.1    ggplot2_3.5.2      dplyr_1.1.4       
[5] data.table_1.17.8  workflowr_1.7.1   

loaded via a namespace (and not attached):
 [1] sass_0.4.10        generics_0.1.4     lattice_0.22-7     stringi_1.8.7     
 [5] digest_0.6.37      magrittr_2.0.3     evaluate_1.0.4     grid_4.5.1        
 [9] RColorBrewer_1.1-3 fastmap_1.2.0      Matrix_1.7-3       R.oo_1.27.1       
[13] rprojroot_2.0.4    jsonlite_2.0.0     processx_3.8.6     R.utils_2.13.0    
[17] whisker_0.4.1      ps_1.9.1           promises_1.3.3     httr_1.4.7        
[21] mgcv_1.9-3         scales_1.4.0       jquerylib_0.1.4    cli_3.6.5         
[25] rlang_1.1.6        R.methodsS3_1.8.2  splines_4.5.1      withr_3.0.2       
[29] cachem_1.1.0       yaml_2.3.10        tools_4.5.1        httpuv_1.6.16     
[33] vctrs_0.6.5        R6_2.6.1           lifecycle_1.0.4    git2r_0.36.2      
[37] stringr_1.5.1      fs_1.6.6           pkgconfig_2.0.3    callr_3.7.6       
[41] pillar_1.11.0      bslib_0.9.0        later_1.4.2        gtable_0.3.6      
[45] glue_1.8.0         Rcpp_1.1.0         xfun_0.52          tibble_3.3.0      
[49] tidyselect_1.2.1   rstudioapi_0.17.1  knitr_1.50         farver_2.1.2      
[53] nlme_3.1-168       htmltools_0.5.8.1  labeling_0.4.3     rmarkdown_2.29    
[57] compiler_4.5.1     getPass_0.2-4     </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
