<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu, Ph.D." />


<title>Compare GWAS results on the same trait from two versions of input data files</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scratch</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Compare GWAS results on the same trait from
two versions of input data files</h1>
<h4 class="author">Xiang Zhu, Ph.D.</h4>
<h4 class="date">First created on 2025-04-04; Last modified on
2025-07-11</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-07-11
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>scratch/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250402code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250402)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250402code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250402)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree850581574fcefff893e943d06de1c1b527ee78aatargetblank8505815a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/scratch/tree/850581574fcefff893e943d06de1c1b527ee78aa" target="_blank">8505815</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhuscratchtree850581574fcefff893e943d06de1c1b527ee78aatargetblank8505815a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/scratch/tree/850581574fcefff893e943d06de1c1b527ee78aa" target="_blank">8505815</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/compare_gcst90019014.Rmd</code>)
and HTML (<code>docs/compare_gcst90019014.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/scratch/blob/850581574fcefff893e943d06de1c1b527ee78aa/analysis/compare_gcst90019014.Rmd" target="_blank">8505815</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-07-11
</td>
<td>
wflow_publish("analysis/compare_gcst90019014.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/scratch/cbb5be3f5c9ce8183f14d03faa344bcce95aa800/docs/compare_gcst90019014.html" target="_blank">cbb5be3</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/scratch/blob/415e24242e79b36e84b3fba7c1d4daaf907cff03/analysis/compare_gcst90019014.Rmd" target="_blank">415e242</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2025-04-14
</td>
<td>
wflow_publish("analysis/compare_gcst90019014.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="ukb-wgs" class="section level2">
<h2>UKB-WGS</h2>
<div id="overview-of-two-gwas-for-comparison" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of rows in the new GWAS:  82465759 </code></pre>
<pre><code>Number of rows in the old GWAS:  16241904 </code></pre>
<pre><code>Number of genome-wide significant variants in the new GWAS:  688 </code></pre>
<pre><code>Number of genome-wide significant variants in the old GWAS:  153 </code></pre>
<pre><code>Number of GWAS loci in the new GWAS:  25 </code></pre>
<pre><code>Number of GWAS loci in the old GWAS:  8 </code></pre>
<pre><code>Number of rows after joining two data frames:  15983783 </code></pre>
<pre><code>Number of rows with the same alleles:  15137872 </code></pre>
<pre><code>Number of rows with flipped alleles:  2819 </code></pre>
<pre><code>Number of rows with inconsistent alleles:  843092 </code></pre>
<pre><code>Number of rows after removing inconsistent alleles:  15140691 </code></pre>
</div>
<div id="comparison-across-all-variants" class="section level3">
<h3>Comparison across all variants</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.30698379 , 95% CI = [ -0.30743995 ,  -0.30652748 ]

Regression coefficients for new_BETA (new ~ old):
                 Estimate   Std. Error     t value      Pr(&gt;|t|)
(Intercept)  0.0005173757 2.431882e-05    21.27471 1.953947e-100
x           -0.1006079961 8.015877e-05 -1255.10905  0.000000e+00

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.25392139 , 95% CI = [ 0.2534501 ,  0.25439256 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.3393383 0.0001665872 2037.000        0
x           0.2644892 0.0002589186 1021.515        0

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.4668177 , 95% CI = [ -0.46721162 ,  -0.4664236 ]

Regression coefficients for new_ZSCORE (new ~ old):
                Estimate   Std. Error      t value     Pr(&gt;|t|)
(Intercept) -0.001000381 0.0002366568    -4.227138 2.366839e-05
x           -0.473429507 0.0002305369 -2053.595168 0.000000e+00</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-new-gwas"
class="section level3">
<h3>Comparison across genome-wide significant variants from new
GWAS</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.97544327 , 95% CI = [ -0.97901207 ,  -0.97127643 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate   Std. Error     t value    Pr(&gt;|t|)
(Intercept)  0.001219415 0.0003695806    3.299455 0.001025154
x           -0.474285270 0.0043360099 -109.382885 0.000000000

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.36610956 , 95% CI = [ 0.2954265 ,  0.43280639 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate Std. Error   t value      Pr(&gt;|t|)
(Intercept) 7.2653170 0.13993125 51.920620 5.028622e-226
x           0.2308061 0.02375315  9.716864  7.558479e-21

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.96901834 , 95% CI = [ -0.97350835 ,  -0.9637813 ]

Regression coefficients for new_ZSCORE (new ~ old):
             Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept)  0.208492 0.06039378   3.45221 0.0005944839
x           -1.262061 0.01302448 -96.89909 0.0000000000</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-old-gwas"
class="section level3">
<h3>Comparison across genome-wide significant variants from old
GWAS</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.96933585 , 95% CI = [ -0.97766441 ,  -0.95796771 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate  Std. Error    t value     Pr(&gt;|t|)
(Intercept)  0.003460711 0.001061676   3.259669 1.380235e-03
x           -0.353530685 0.007317835 -48.310833 2.499309e-93

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  -0.0929617 , 95% CI = [ -0.24848493 ,  0.06723404 ]

Regression coefficients for new_LOG10P (new ~ old):
              Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 11.8090070  1.7824939  6.624992 5.842389e-10
x           -0.2414711  0.2111693 -1.143495 2.546546e-01

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.99279175 , 95% CI = [ -0.99476649 ,  -0.99007562 ]

Regression coefficients for new_ZSCORE (new ~ old):
              Estimate Std. Error     t value      Pr(&gt;|t|)
(Intercept)  0.3057624 0.06284421    4.865404  2.860605e-06
x           -1.0887702 0.01073194 -101.451404 4.103271e-140</code></pre>
</div>
<div id="overlap-of-genome-wide-significant-variants-between-two-gwas"
class="section level3">
<h3>Overlap of genome-wide significant variants between two GWAS</h3>
<pre><code>                     New GWAS p &lt; cutoff New GWAS p &gt;= cutoff
Old GWAS p &lt; cutoff                  140                   12
Old GWAS p &gt;= cutoff                 472             15140067</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value &lt; 2.2e-16
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 8.402051e+03 4.503600e+15
sample estimates:
odds ratio 
4.5036e+15 </code></pre>
</div>
<div id="visualization-of-comparison" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_gcst90019014.Rmd/unnamed-chunk-4-1.png" width="1440" style="display: block; margin: auto;" /><img src="figure/compare_gcst90019014.Rmd/unnamed-chunk-4-2.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ukb-gel" class="section level2">
<h2>UKB-GEL</h2>
<div id="overview-of-two-gwas-for-comparison-1" class="section level3">
<h3>Overview of two GWAS for comparison</h3>
<pre><code>Number of rows in the new GWAS:  53594832 </code></pre>
<pre><code>Number of rows in the old GWAS:  16241904 </code></pre>
<pre><code>Number of genome-wide significant variants in the new GWAS:  352 </code></pre>
<pre><code>Number of genome-wide significant variants in the old GWAS:  153 </code></pre>
<pre><code>Number of GWAS loci in the new GWAS:  20 </code></pre>
<pre><code>Number of GWAS loci in the old GWAS:  8 </code></pre>
<pre><code>Relaxed genome-wide significant threshold:  7.30103 </code></pre>
<pre><code>Number of rows after joining two data frames:  14605801 </code></pre>
<pre><code>Number of rows with the same alleles:  14381537 </code></pre>
<pre><code>Number of rows with flipped alleles:  1440 </code></pre>
<pre><code>Number of rows with inconsistent alleles:  222824 </code></pre>
<pre><code>Number of rows after removing inconsistent alleles:  14382977 </code></pre>
</div>
<div id="comparison-across-all-variants-1" class="section level3">
<h3>Comparison across all variants</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.31330711 , 95% CI = [ -0.3137731 ,  -0.31284096 ]

Regression coefficients for new_BETA (new ~ old):
                 Estimate   Std. Error     t value      Pr(&gt;|t|)
(Intercept) -0.0006124382 2.419074e-05   -25.31705 2.088119e-141
x           -0.1167655625 9.332212e-05 -1251.20994  0.000000e+00

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.25767622 , 95% CI = [ 0.25719367 ,  0.25815864 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate   Std. Error  t value Pr(&gt;|t|)
(Intercept) 0.3363164 0.0001687797 1992.636        0
x           0.2654675 0.0002624785 1011.388        0

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.47386156 , 95% CI = [ -0.47426225 ,  -0.47346068 ]

Regression coefficients for new_ZSCORE (new ~ old):
                Estimate   Std. Error     t value      Pr(&gt;|t|)
(Intercept) -0.006437118 0.0002407413   -26.73873 1.684815e-157
x           -0.478525643 0.0002344994 -2040.62602  0.000000e+00</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-new-gwas-1"
class="section level3">
<h3>Comparison across genome-wide significant variants from new
GWAS</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.97701595 , 95% CI = [ -0.98143555 ,  -0.97155929 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate   Std. Error    t value      Pr(&gt;|t|)
(Intercept)  0.000554842 0.0005339984   1.039033  2.995478e-01
x           -0.497930450 0.0059713341 -83.386801 2.867508e-224

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  0.64739524 , 95% CI = [ 0.58029608 ,  0.70576441 ]

Regression coefficients for new_LOG10P (new ~ old):
             Estimate Std. Error  t value      Pr(&gt;|t|)
(Intercept) 6.8152696 0.13537264 50.34451 3.398559e-157
x           0.3412524 0.02208184 15.45399  6.094923e-41

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.97209607 , 95% CI = [ -0.97745092 ,  -0.96549176 ]

Regression coefficients for new_ZSCORE (new ~ old):
              Estimate Std. Error    t value      Pr(&gt;|t|)
(Intercept)  0.1535675 0.07789113   1.971566  4.949170e-02
x           -1.2510019 0.01659325 -75.392206 1.667753e-210</code></pre>
</div>
<div
id="comparison-across-genome-wide-significant-variants-from-old-gwas-1"
class="section level3">
<h3>Comparison across genome-wide significant variants from old
GWAS</h3>
<pre><code>Pearson correlation between new_BETA in new and old_BETA in old:
Estimate =  -0.98224702 , 95% CI = [ -0.98713385 ,  -0.97552694 ]

Regression coefficients for new_BETA (new ~ old):
                Estimate  Std. Error    t value      Pr(&gt;|t|)
(Intercept)  0.003306224 0.000886861   3.728007  2.747741e-04
x           -0.386458932 0.006087493 -63.484086 9.721875e-109

Pearson correlation between new_LOG10P in new and old_LOG10P in old:
Estimate =  -0.20734325 , 95% CI = [ -0.35626505 ,  -0.04814861 ]

Regression coefficients for new_LOG10P (new ~ old):
              Estimate Std. Error   t value     Pr(&gt;|t|)
(Intercept) 13.6628860  1.6530444  8.265287 7.544457e-14
x           -0.5027028  0.1956233 -2.569748 1.117216e-02

Pearson correlation between new_ZSCORE in new and old_ZSCORE in old:
Estimate =  -0.99198738 , 95% CI = [ -0.99420084 ,  -0.98893375 ]

Regression coefficients for new_ZSCORE (new ~ old):
              Estimate Std. Error    t value      Pr(&gt;|t|)
(Intercept)  0.2765292 0.06555945   4.217991  4.291297e-05
x           -1.0652881 0.01119007 -95.199381 5.571865e-134</code></pre>
</div>
<div id="overlap-of-genome-wide-significant-variants-between-two-gwas-1"
class="section level3">
<h3>Overlap of genome-wide significant variants between two GWAS</h3>
<pre><code>                     New GWAS p &lt; cutoff New GWAS p &gt;= cutoff
Old GWAS p &lt; cutoff                  126                   23
Old GWAS p &gt;= cutoff                 207             14382621</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table
p-value &lt; 2.2e-16
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 8.402051e+03 4.503600e+15
sample estimates:
odds ratio 
4.5036e+15 </code></pre>
</div>
<div id="visualization-of-comparison-1" class="section level3">
<h3>Visualization of comparison</h3>
<p><img src="figure/compare_gcst90019014.Rmd/unnamed-chunk-7-1.png" width="1440" style="display: block; margin: auto;" /><img src="figure/compare_gcst90019014.Rmd/unnamed-chunk-7-2.png" width="1440" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.5.1 (2025-06-13)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.5-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.1

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Los_Angeles
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] MungeSumstats_1.16.0 xzTools_0.0.0.9000   patchwork_1.3.1     
[4] ggplot2_3.5.2        dplyr_1.1.4          data.table_1.17.8   
[7] workflowr_1.7.1     

loaded via a namespace (and not attached):
  [1] DBI_1.2.3                   bitops_1.0-9               
  [3] rlang_1.1.6                 magrittr_2.0.3             
  [5] git2r_0.36.2                matrixStats_1.5.0          
  [7] compiler_4.5.1              RSQLite_2.4.1              
  [9] getPass_0.2-4               mgcv_1.9-3                 
 [11] GenomicFeatures_1.60.0      png_0.1-8                  
 [13] callr_3.7.6                 vctrs_0.6.5                
 [15] stringr_1.5.1               pkgconfig_2.0.3            
 [17] crayon_1.5.3                fastmap_1.2.0              
 [19] XVector_0.48.0              labeling_0.4.3             
 [21] Rsamtools_2.24.0            promises_1.3.3             
 [23] rmarkdown_2.29              UCSC.utils_1.4.0           
 [25] ps_1.9.1                    bit_4.6.0                  
 [27] xfun_0.52                   cachem_1.1.0               
 [29] GenomeInfoDb_1.44.0         jsonlite_2.0.0             
 [31] blob_1.2.4                  later_1.4.2                
 [33] DelayedArray_0.34.1         BiocParallel_1.42.1        
 [35] parallel_4.5.1              R6_2.6.1                   
 [37] VariantAnnotation_1.54.1    bslib_0.9.0                
 [39] stringi_1.8.7               RColorBrewer_1.1-3         
 [41] rtracklayer_1.68.0          GenomicRanges_1.60.0       
 [43] jquerylib_0.1.4             Rcpp_1.1.0                 
 [45] SummarizedExperiment_1.38.1 knitr_1.50                 
 [47] R.utils_2.13.0              IRanges_2.42.0             
 [49] httpuv_1.6.16               Matrix_1.7-3               
 [51] splines_4.5.1               tidyselect_1.2.1           
 [53] rstudioapi_0.17.1           abind_1.4-8                
 [55] yaml_2.3.10                 codetools_0.2-20           
 [57] curl_6.4.0                  processx_3.8.6             
 [59] lattice_0.22-7              tibble_3.3.0               
 [61] Biobase_2.68.0              withr_3.0.2                
 [63] KEGGREST_1.48.1             evaluate_1.0.4             
 [65] Biostrings_2.76.0           pillar_1.11.0              
 [67] MatrixGenerics_1.20.0       whisker_0.4.1              
 [69] stats4_4.5.1                ieugwasr_1.0.4             
 [71] generics_0.1.4              rprojroot_2.0.4            
 [73] RCurl_1.98-1.17             S4Vectors_0.46.0           
 [75] scales_1.4.0                glue_1.8.0                 
 [77] tools_4.5.1                 BiocIO_1.18.0              
 [79] BSgenome_1.76.0             GenomicAlignments_1.44.0   
 [81] fs_1.6.6                    XML_3.99-0.18              
 [83] grid_4.5.1                  AnnotationDbi_1.70.0       
 [85] nlme_3.1-168                GenomeInfoDbData_1.2.14    
 [87] restfulr_0.0.16             cli_3.6.5                  
 [89] S4Arrays_1.8.1              gtable_0.3.6               
 [91] R.methodsS3_1.8.2           sass_0.4.10                
 [93] digest_0.6.37               BiocGenerics_0.54.0        
 [95] SparseArray_1.8.0           rjson_0.2.23               
 [97] farver_2.1.2                memoise_2.0.1              
 [99] htmltools_0.5.8.1           R.oo_1.27.1                
[101] lifecycle_1.0.4             httr_1.4.7                 
[103] bit64_4.6.0-1              </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
